portfolio/src/main/webapp/index.html

```diff
@@ -85,11 +85,15 @@ <h3>Projects</h3>
 
       <div id="form-container">
         <form id="my-form" class="hidden" method="POST" 
-              enctype="multipart/form-data">
+              enctype="multipart/form-data" novalidate>
```

Comment:
why do you need ```novalidate```?

Reply:
I got "An invalid form control with name…is not focusable" error without the "novalidate".
Seems like this is because I make latitude and longitude hidden elements in the form but they are labelled as required by default.

Reply:
Could we solve this issue by providing them empty values?

Reply:
I got the same not focusable error when setting value="", I also tried value="0", it does not work as well.

Reply:
I would suggest 
1. first try to remove the hidden to see which field cause the failures.
2. If it's caused by missing required fields, try
```
document.getElementById('field_id').removeAttribute("required");
```

Reply:
I tried this statement and also `document.getElementById('field_id').required = false;`, but still got the not focusable error.
I'm wondering if I should not include hidden elements in the form?

Reply:
Approve to unblock,
But I think you could reach to virtual support. ```novalidate``` makes it won't validate all fields.

Reply:
Okay! Yes, I think there should be some way better than novalidate.